File: 2KD/src/main/java/com/example/a2kd/MainActivity.java
package com.example.a2kd;

import android.content.ComponentName;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Switch;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private Switch sw;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        sw = findViewById(R.id.swAlarm);

        sw.setOnCheckedChangeListener((buttonView, isChecked) -> {
            if (isChecked) {
                Intent i = new Intent();

                i.setComponent(new ComponentName(
                        "com.example.a2kdpart2",
                        "com.example.a2kdpart2.MainActivity"
                ));
                startActivity(i);

                sw.setChecked(false);
            }
        });
    }
}

File: 2LD/src/main/java/com/example/a2ld/MainActivity.kt
package com.example.a2ld

import android.os.Bundle
import android.widget.Button // mygtukų biblioteka
import android.widget.TextView // teksto biblioteka
import android.util.Log //
import android.view.View //
import androidx.appcompat.app.AppCompatActivity
class MainActivity : AppCompatActivity() {

    Button button1, button2, button3;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
